projects/ion/src/lib/chip/chip.component.ts

```diff
@@ -4,6 +4,7 @@ import { DropdownItem } from '../dropdown/dropdown.component';
 import { InfoBadgeStatus } from '../core/types';
 
 export type ChipSize = 'sm' | 'md';
+export type Direction = 'right' | 'left';
```

Comment:
```suggestion
export type IconDirection = 'right' | 'left';
```

I believe that make more sense

Reply:
or use your attribute name, IconPosition.